src/print-prescription/prescription-printout.component.tsx

```diff
@@ -0,0 +1,121 @@
+              return (
+                <div key={index}>
+                  {dosageInstruction && (
+                    <StructuredListRow>
+                      <StructuredListCell>
+                        <p>
+                          <strong>
+                            {getMedicationDisplay(getMedicationReferenceOrCodeableConcept(medicationEvent))}
+                          </strong>
+                        </p>
+                        <br />
+                        <p>
+                          <span className={styles.faintText}>{t('dose', 'Dose')}</span>
+                          {': '}
+                          <span>
+                            {dosageInstruction?.doseAndRate?.map((doseAndRate, index) => {
+                              return (
+                                <span key={index}>
+                                  {doseAndRate?.doseQuantity?.value} {doseAndRate?.doseQuantity?.unit}
+                                </span>
+                              );
+                            })}
+                          </span>{' '}
+                          &mdash; {dosageInstruction?.route?.text} &mdash; {dosageInstruction?.timing?.code?.text}{' '}
+                          {dosageInstruction?.timing?.repeat?.duration
+                            ? 'for ' +
+                              dosageInstruction?.timing?.repeat?.duration +
+                              ' ' +
+                              dosageInstruction?.timing?.repeat?.durationUnit
+                            : ' '}
+                          {quantity && (
+                            <p>
+                              <span className={styles.faintText}>{t('quantity', 'Quantity')}</span>
+                              {': '}
+                              <span>
+                                {quantity.value} {quantity.unit}
+                              </span>
+                            </p>
+                          )}
+                        </p>
+                        <p>
+                          <span className={styles.faintText}>{t('datePrescribed', 'Date prescribed')}</span>
+                          {': '} <span>{formatDate(parseDate((request.request as any).authoredOn))}</span>
+                        </p>
+                        {(refillsAllowed || refillsAllowed === 0) && (
+                          <p>
+                            <span className={styles.faintText}>{t('refills', 'Refills')}</span>
+                            {': '} <span>{refillsAllowed}</span>
+                          </p>
+                        )}
+                        {dosageInstruction?.text && <p>{dosageInstruction.text}</p>}
+                        {dosageInstruction?.additionalInstruction?.length > 0 && (
+                          <p>{dosageInstruction?.additionalInstruction[0].text}</p>
+                        )}
+                      </StructuredListCell>
+                    </StructuredListRow>
```

Comment:
I just scanned this quickly, but during testing we should make sure that this properly displays all the fields we want it to display.